ant-node/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 authors = ["MaidSafe Developers <dev@maidsafe.net>"]
 description = "The Autonomi node binary"
 name = "ant-node"
-version = "0.3.1"
+version = "0.3.2"
```

Comment:
I think the version bumping should be left for the release process, so I would revert this.